service-api/src/main/java/greencity/dto/AddNewTariffDto.java

```diff
@@ -18,6 +19,7 @@ public class AddNewTariffDto {
     @NotNull
     @Min(1)
     private Long courierId;
+    @NotEmpty
     @NotNull
     private List<@Min(1) Long> locationIdList;
```

Comment:
Why do we need `@NotEmpty` annotation here? Does `@Min(1)` not work? If it does not work please remove it.

Reply:
Oh, I have an idea.
@Min(1) work correctly. It check's that values of our ids in locationIdList are equal or greather than 1.
I added @NonEmpty for checking that our methods receive non empty locationIdList as a parametr. So, I think, after adding this annotation we don't need the test which we discused yesterday, so we can remove such checking "if (locationSet.isEmpty())" from our private methods in SuperAdminServiceImpl class. 

Can I send pool reaqust with such chang?